systemd/nexus-init.sh

```diff
@@ -43,14 +43,9 @@ NEXUS_VOLUME_MOUNT="/nexus-data:rw,exec"
 
 # Create Nexus volume if not already present
 if ! podman volume inspect "$NEXUS_VOLUME_NAME" &>/dev/null; then
-    # Load busybox image if it doesn't already exist
+    # Load nexus image if it doesn't already exist
     if ! podman image inspect "$NEXUS_IMAGE" &>/dev/null; then
-        # load the image
         podman load -i "$NEXUS_IMAGE_PATH" || exit
-        # get the tag
-        NEXUS_IMAGE_ID=$(podman images --noheading --format "{{.Id}}" --filter label="name=Nexus Repository Manager")
```

Comment:
This may not work properly, if there are more then 1 nexus images in podman. Anyway it's not needed anymore, because image comes already tagged after change in packaging.